src/assets/scripts/aircraft/AircraftController.js

```diff
@@ -359,6 +347,7 @@ export default class AircraftController {
         return this.aircraft.models[icao];
     }
 
+    // TODO: what is an `eid` and why would it beed to be updated?
```

Comment:
It's my moronic way to try to create unique IDs for individual aircraft so they can be distinguished from each other when talking back and forth between `Aircraft` and `AircraftFlightManagamentSystem` without the need for the existing circular dependencies. (points for trying to remove them, despite the lack of a good solution 😉)

So it's like the `_id` that all aircraft will inherit from `BaseModel`. It was meant as a way to distinguish, and if replaced with another method to do this, the EIDs can be removed. 

Fun fact: it's for "entity ID", the verbage used in the simulation software I helped with when I worked for BGI on their [MFOQA](http://www.bgi-llc.com/content/textblock/Analysis-and-Reporting-Client-ARC/) product for the US Navy. ✈️